src/components/common/YourCompany/YourCompany.vue

```diff
@@ -451,32 +450,26 @@ export default class YourCompany extends Mixins(
   /** The recognition/founding (aka effective or start date) datetime. */
   get recognitionDateTime (): string {
     if (this.isBenIaCorrectionFiling) {
-      if (this.getOriginalEffectiveDateTime) {
-        return (this.apiToPacificDateLong(this.getOriginalEffectiveDateTime))
+      if (this.getCorrectedFilingDate) {
+        return this.apiToPacificDateTime(this.getCorrectedFilingDate)
       }
     }
     if (this.isFirmCorrectionFiling) {
       if (this.getBusinessFoundingDate) {
-        return (this.apiToPacificDateLong(this.getBusinessFoundingDate))
+        return this.apiToPacificDateTime(this.getBusinessFoundingDate)
       }
     }
     if (this.isAlterationFiling || this.isChangeRegFiling || this.isFirmConversionFiling) {
       if (this.getBusinessFoundingDate) {
-        return (this.apiToPacificDateLong(this.getBusinessFoundingDate))
+        return this.apiToPacificDateLong(this.getBusinessFoundingDate)
```

Comment:
This is supposed to be "recognition date time" but this returns only the date.

I fixed the 2 corrections because we're working on them currently. I didn't want to change the other filings since we're not looking at them.

Reply:
This one didn't work either so I fixed it same as the other 2.